bin/local-env/includes.sh

```diff
@@ -158,7 +158,7 @@ command_exists() {
 # Calls docker-compose with common options.
 ##
 dc() {
-	docker-compose $DOCKER_COMPOSE_FILE_OPTIONS "$@"
+	docker compose $DOCKER_COMPOSE_FILE_OPTIONS "$@"
```

Comment:
This is amazingly the only real change that seems required for us to migrate to Compose v2.